grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientTest.java

```diff
@@ -211,11 +211,9 @@ void setUp() {
         final URI uri = server.httpUri(GrpcSerializationFormats.PROTO);
         blockingStub = Clients.builder(uri)
                               .maxResponseLength(MAX_MESSAGE_SIZE)
-                              .decorator(LoggingClient.builder().newDecorator())
```

Comment:
Maybe better reverting this? Is it too noisy?

Reply:
Yes, and it makes the IntelliJ stuck. 😱

Reply:
Or set the log level to `TRACE`?

Reply:
Let me just remove this. If we need if for debugging we can easily add it. 😄